test-support/page-object.js

```diff
@@ -58,3 +58,5 @@ export default {
   value,
   visitable
 };
+
+console.warn("DEPRECATION: Importing ember-cli-page-object from your app's `tests/` folder has been deprecated. Instead, import from an absolute path like 'ember-cli-page-object', 'ember-cli-page-object/create', etc.");
```

Comment:
Don't print a deprecation warning for now, let's add the feature and see how it goes.